masterdata/returnRequest/schema.json

```diff
@@ -0,0 +1,211 @@
+{
+  "properties": {
+    "orderId": {
+      "type": "string",
+      "required": true
+    },
+    "totalReturnAmount": {
+      "type": "integer",
+      "required": true
+    },
+    "sequenceNumber": {
+      "type": "string",
+      "required": true
+    },
+    "status": { "$ref": "#/$defs/status", "required": true },
+    "returnTotals": {
+      "type": "array",
+      "required": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "required": true,
+            "enum": ["items", "shipping", "tax"]
+          },
+          "value": {
+            "type": "integer",
+            "required": true
+          }
+        }
+      }
+    },
+    "customerProfileData": {
+      "type": "object",
+      "required": true,
+      "properties": {
+        "userId": {
+          "type": "string",
+          "required": true
+        },
+        "name": {
+          "type": "string"
+        },
+        "email": {
+          "type": "string",
+          "format": "email"
+        },
+        "phoneNumber": { "type": "string", "maxLength": 50 }
+      }
+    },
```

Comment:
The `customerProfileData` groups the user info data. We could store only the `userId` and fetch user info from CL (to be PII complaint) every time we need it, but than we should handle:
1. User will not be able to change name and phone (as it can do it in the current version);
2. It wouldn't be fully backward compatible, since we would lose the information for the cases that the user has changed the name and phone number;
3. How to handle Marketplace and Seller? Does the info related to customer lives on Marketplace or seller? Does the Markeplace has full access to CL from Seller?

Reply:
for marketplace seller, the current approach is the best. accounts cannot read external master datas, so a seller won't be able to access the marketplace nor vice versa unless there is a custom service allowing that